kubernetes-workshop/8004_network/modul_network_8004_01_slides.md

```diff
@@ -55,6 +60,14 @@ Introduction to the networking layers in Kubernetes
 
 ![](static/kubernetes_service_network.png "Kubernetes ClusterIP Network Flow")
 
+##
+
+* Traffic from client process lands on pod veth0
+* Gets routed to a different node, through kube-proxy if needed
```

Comment:
```suggestion
* Get's picked up by iptables chain for that service
* Service ip maps to DNAT for target Pod ip(s)
* Traffic is forwarded to Pod ip
```
Multi-line suggestions don't seem to work that well, but I guess you get the idea. The service chain in iptables is managed by kube-proxy on every node in the cluster.

Reply:
> but I guess you get the idea

Not really :wink: 

Do we really want to go to the level of detail in you suggestion?

Reply:
We don't have to, but kube-proxy maintains the same iptables configuration on every node, so at least that should be fixed.